app/SharedWebComponents/Components/VoiceDialog.razor.cs

```diff
@@ -21,7 +21,16 @@ protected override async Task OnInitializedAsync()
         _state = RequestVoiceState.RequestingVoices;
 
         await GetVoicesAsync();
-        SpeechSynthesis.OnVoicesChanged(() => GetVoicesAsync(true));
+
+        try
+        {
+            SpeechSynthesis.OnVoicesChanged(() => GetVoicesAsync(true));
+        }
+        catch
+        {
+            // do nothing if the service does not support observation
```

Comment:
I'm not sure I follow this comment, what's the concern this addresses?

Reply:
Added a much better comment.